django/db/models/expressions.py

```diff
@@ -390,6 +390,18 @@ def relabeled_clone(self, change_map):
     def copy(self):
         return copy.copy(self)
 
+    def prefix_references(self, prefix):
+        clone = self.copy()
+        clone.set_source_expressions(
+            [
+                F(f"{prefix}__{expr.name}")
```

Comment:
Minor but I think the `LOOKUP_SEP` should not be baked in `prefix_references` to allow caller to include it themselves.

```suggestion
                F(f"{prefix}{expr.name}")
```